plugin-name/plugin-name.php

```diff
@@ -66,10 +59,5 @@ function deactivate_plugin_name() {
  *
  * @since    1.0.0
  */
-function run_plugin_name() {
-
-	$plugin = new Plugin_Name();
-	$plugin->run();
-
-}
-run_plugin_name();
+$plugin = new Plugin_Name();
```

Comment:
I'm not in charge here; I just figured, since you proposed it, you thought it was better. My apologies; I did mean for that to come off so terse.